src/components/Footer.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { COLOUR_PRIMARY_BACKGROUND_ALTERNATIVE } from '../theme'
+import ConstrainedWidth from './Layout/ConstrainedWidth'
+import Text from './Text'
+import { ExternalLink } from './Link'
+import Heading from './Heading'
+
+const HousekeepingList = ({ children, title }) => (
```

Comment:
Perhaps `children` should be made to be a required field. We're using the component several times without specifying children, which results in an empty `<ul>`. Bonus points to validating children as `li` elements :stuck_out_tongue_closed_eyes: